source/CommandLine/Extensions/AssemblyExtensions.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace Octopus.CommandLine.Extensions
+{
+    public static class AssemblyExtensions
+    {
+        public static string GetExecutablePath()
+        {
+            return Process.GetCurrentProcess()?.MainModule?.FileName ?? throw new ApplicationException("Unable to determine executable name");
```

Comment:
This uses the new net5(?) way of getting it, whereas we were using the deprecated `assembly.Codebase` previously.